tools/aie2xclbin/XCLBinGen.cpp

```diff
@@ -299,17 +299,13 @@ static LogicalResult generateCoreElfFiles(ModuleOp moduleOp,
         std::string targetLower = StringRef(TK.TargetArch).lower();
         SmallVector<std::string, 10> flags;
         flags.push_back("-O2");
-        std::string targetFlag = "--target=" + targetLower + "-none-elf";
+        std::string targetFlag = "--target=" + targetLower;
         flags.push_back(targetFlag);
         flags.emplace_back(objFile);
         SmallString<64> meBasicPath(TK.InstallDir);
         sys::path::append(meBasicPath, "aie_runtime_lib", TK.TargetArch,
                           "me_basic.o");
         flags.emplace_back(meBasicPath);
-        SmallString<64> libcPath(TK.PeanoDir);
```

Comment:
perhaps ifdef windows would be appropriate here, so we can still have full functionality on Linux?